src/test/java/gregtech/api/recipes/logic/IParallelableRecipeLogicTest.java

```diff
@@ -713,6 +714,45 @@ public MetaTileEntity getMetaTileEntity() {
         MatcherAssert.assertThat(outputRecipe, nullValue());
     }
 
+    @Test
+    public void findParallelRecipe_SmallMaxStackSize() {
+        MetaTileEntityElectricBlastFurnace EBF = initEBF(521);
+
+        int parallelLimit = 4;
+
+        // Create a recipe Map to be used for testing
+        RecipeMap<BlastRecipeBuilder> map = new RecipeMapBuilder<>("electric_blast_furnace", new BlastRecipeBuilder())
+                .itemInputs(3).itemOutputs(2).fluidInputs(1).fluidOutputs(1).build();
+
+        // Create a simple recipe to be used for testing
+        // Use an output item with small max stack size
+        Recipe recipe = map.recipeBuilder()
+                .inputs(new ItemStack(Blocks.COBBLESTONE))
+                .outputs(new ItemStack(Items.ELYTRA))
+                .blastFurnaceTemp(1000)
+                .EUt(30).duration(100)
+                .build().getResult();
+
+        IParallelableRecipeLogic logic = new ParallelableTestLogic(EBF, map, ParallelLogicType.MULTIPLY);
+
+        // Populate the Input Bus
+        importItemBus.getImportItems().insertItem(0, new ItemStack(Blocks.COBBLESTONE, 16), false);
+
+        // Saturate the export bus, except for one slot
+        exportItemBus.getExportItems().insertItem(0, new ItemStack(Blocks.BONE_BLOCK, 16), false);
+        exportItemBus.getExportItems().insertItem(1, new ItemStack(Blocks.BONE_BLOCK, 16), false);
+        exportItemBus.getExportItems().insertItem(2, new ItemStack(Blocks.BONE_BLOCK, 16), false);
+        // exportItemBus.getExportItems().insertItem(3, new ItemStack(Blocks.BONE_BLOCK, 16), false);
```

Comment:
This commented out code should be removed.

Reply:
Fixed in new commit.